api/swap/approval/_utils.ts

```diff
@@ -326,9 +338,16 @@ async function _buildDepositTxForAllowanceHolderSvm(
     sdk.utils.getCurrentTime() + getFillDeadlineBuffer(destinationChainId);
   const exclusivityParameter =
     crossSwapQuotes.bridgeQuote.suggestedFees.exclusivityDeadline;
+  // TODO: Bug - message handling is broken and always uses empty messages
+  // Cannot fix yet because including actual messages would make transactions too large
+  // Future implementation should use: crossSwapQuotes.bridgeQuote.message?.slice(2) ?? "0x"
   const message = Uint8Array.from(
     Buffer.from(crossSwapQuotes.bridgeQuote.message ?? "0x", "hex")
```

Comment:
Does this work as is? I remember I needed to remove the 0x regardless while testing.
```suggestion
    Buffer.from((crossSwapQuotes.bridgeQuote.message ?? "0x").slice(2), "hex")
```